machina/locale/fr/LC_MESSAGES/django.po

```diff
@@ -64,7 +64,7 @@ msgid "Direct number of posts"
 msgstr "Nombre direct de messages"
 
 msgid "Direct number of topics"
-msgstr "Nombre direct de sujets"
+msgstr "Nombre direct de thèmes"
```

Comment:
I think "Sujet" is more frequent than "Thème" when it comes to name forum discussions. Could we keep using "Sujet"? 🙂